src/libopenrave/planner.cpp

```diff
@@ -99,18 +99,12 @@ int AddStatesWithLimitCheck(std::vector<dReal>& q, const std::vector<dReal>& qde
     return status;
 }
 
-PlannerStatus::PlannerStatus() : statusCode(0)
-{
-}
+PlannerStatus::PlannerStatus() {}
```

Comment:
By removing `statusCode(0)` you introduced a bug. Can you revert to exactly what it was before here?